modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageLayoutEditorDisplayContext.java

```diff
@@ -150,7 +198,7 @@ public Map<String, Object> getEditorContext(String npmResolvedPackageName)
 
 		LearnMessage learnMessage = LearnMessageUtil.getLearnMessage(
 			"content-page-personalization",
-			LanguageUtil.getLanguageId(httpServletRequest),
+			super.language.getLanguageId(httpServletRequest),
```

Comment:
Why do you need to use the super?

Reply:
I think that it is more clear to know that this var it is saved in parent class. Is it a bad praxis?